src/move/emojicoin_dot_fun/sources/emojicoin_dot_fun.move

```diff
@@ -601,7 +618,7 @@ module emojicoin_dot_fun::emojicoin_dot_fun {
                 base_volume: 0,
                 quote_volume: 0,
                 nonce: 0,
-                time,
+                time: 0,
```

Comment:
so this is `0` because `last_bump_time` records the current time now, right? What exactly does `time` track now?

Reply:
This field tracks the time of the last swap

It initializes to 0, basically as a null value 

(it would be more expensive to use an option here, though that might be slightly more intuitive)

Reply:
Ahh okay, no that makes sense, just was confused why it went from `time` to `0`